src/_pytest/capture.py

```diff
@@ -136,41 +136,59 @@ def suspend_global_capture(self, in_=False):
 
     def suspend(self, in_=False):
         # Need to undo local capsys-et-al if it exists before disabling global capture.
-        self.suspend_fixture(self._current_item)
+        self.suspend_fixture()
         self.suspend_global_capture(in_)
 
     def resume(self):
         self.resume_global_capture()
-        self.resume_fixture(self._current_item)
+        self.resume_fixture()
 
     def read_global_capture(self):
         return self._global_capturing.readouterr()
 
     # Fixture Control (it's just forwarding, think about removing this later)
 
-    def activate_fixture(self, item):
+    @contextlib.contextmanager
+    def _item_fixture(
```

Comment:
Please add a docstring here.

Reply:
Sorry, out of scope.. :)

Reply:
Really?

Reply:
It seemed like a reasonable request (of course it should contain a docstring in the first place already), but OK.

Reply:
i'm under the impression this simple request could at least have been handled more graceful - that looks very frustrating to me

Reply:
??
@nicoddemus @RonnyPfannschmidt 
Just do it yourself then.

Reply:
@RonnyPfannschmidt 
JFI: seeing/getting a comment like this is also not "not frustrating".
To summarize: I've cleaned up code, explicitly stated that adding docs also is out of scope, it got approved again, and then you say this frustrates you.

For the context: this and #6671 were just byproducts when looking at the capture code.
If you would really require to also add docs etc I'll handle this like when adding `__repr__`s, and will just keep it in my "fork" then.

Reply:
But to be clear: if you like for something to get fixed, i.e. have a docstring, then just do it yourself.

Reply:
@RonnyPfannschmidt 
And to be even more clear: you should be aware of me going the extra mile to cover previously uncovered code when refactoring, so your "this is frustrating" comment really annoys me.
I therefore expect you to apologize for it.

Reply:
@blueyed this rant is frankly... bizarre.

All I asked was for you to add a docstring to a method you have changed. Of course the method could have a docstring in the first place, but that's how things go: we strive to improve the code we change. So I just made a suggestion to include a docstring. 

So you went into the code, applied two changes that I made, and instead of spending 10s adding a docstring, you deliberetely decided to not do that, come back here and comment that you chose to not do that, and leaving a comment meaning that this is out of the scope of the PR.

To be clear: when I say "please add a docstring" I don't mean "shame on you, you forgot a docstring here!". I just see an opportunity to improve things, and suggest that. What baffles me is that you took the effort to apply 2 suggestions and leave one out (which again would take you far less time than write all these comments that you made), and get into the PR to comment that you intentionally left them out.

Again, this is bizarre. I don't even know what to make of it, really.

Reply:
And "do it yourself then"... not sure how to take comment on that, frankly.

Reply:
I'm done with that kind of interaction

Reply:
> @RonnyPfannschmidt
> And to be even more clear: you should be aware of me going the extra mile to cover previously uncovered code when refactoring, so your "this is frustrating" comment really annoys me.
> I therefore expect you to apologize for it.

I guess @blueyed has found out that their demand for an apology isn't much appreciated by the community. Why someone should be required to apologise for an expression of frustration simply because code coverage has been increased is quite beyond me.

I'm sorry to see a would-be contributor to open source behaving in a way apparently calculated to be destructive of developer relationships. The main advantage open source has over commercial development is the fact that people do it because it needs doing rather than to make money or score points. I don't understand what the motivation behind these behaviours was, but I hope they will be transformed into more productive activity in future.